src/psyclone/psyir/nodes/reference.py

```diff
@@ -125,7 +125,18 @@ def reference_accesses(self, var_accesses):
             information about variable accesses.
         :type var_accesses: \
             :py:class:`psyclone.core.access_info.VariablesAccessInfo`
+
         '''
+        from psyclone.psyir.nodes import Operation, BinaryOperation
```

Comment:
pylint complains that BinaryOperation is being re-imported here.

Reply:
Fixed.